panel.js

```diff
@@ -221,107 +267,23 @@ function copyToClipboardHandler() {
    }, 1000 * 2);
 }
 
-// add card to favourites
-function makeCardFav(cardnumber) {
-
-  // find card number and mark as fav
-  for (let i = 0; i < cards.length; i++) {
-    let items = cards[i].items;
-
-    for (let j = 0; j < items.length; j++) {
-      let item = items[j];
-      if (item.cardnumber === cardnumber) {
-        item.favourite = true;
-      }
-    }
-  }
-
-  // save to storage and reload
-  setInStorage(CARDS_LIST, cards);
-  load();
-}
-
-// remove card from favourites
-function makeCardUnfav(cardnumber) {
-
-  // find card number and mark as not fav  
-  for (let i = 0; i < cards.length; i++) {
-    let items = cards[i].items;
-
-    for (let j = 0; j < items.length; j++) {
-      let item = items[j];
-      if (item.cardnumber === cardnumber) {
-        item.favourite = false;
-      }
-    }
+// add to favourites
+function makeFav(key) {
```

Comment:
nit: rename `makeFav`/`makeUnfav` functions to `createFavourite`/`removeFavourite` or `addFavourite`/`removeFavourite` 

Something to consider in the future: use a vertical slice architecture (group by actions that users can take) for each functionality